config/renovate/renovate.json

```diff
@@ -93,6 +93,9 @@
         "recreateWhen": "always",
         "rebaseWhen": "behind-base-branch"
       }
+    ],
+    "schedule": [
+      "after 5am on saturday"
```

Comment:
Do you know if we can apply `updateNotScheduled` to tekton manager only? Updates to existing PRs can also trigger PLRs, which is probably not what users want.

Reply:
Yes it can be used inside `packageRules` to tekton only

Reply:
I've rebase the PR. Would it be useful in all the managers?